psql2mysql/__init__.py

```diff
@@ -504,6 +504,10 @@ def main():
         print("Batch processing done.")
         sys.exit(0)
 
+    if not cfg.CONF.source:
+        print("Source database was not specified.")
+        sys.exit(1)
```

Comment:
NIT: we could use different >0 return codes to represent different errors. But it's only a NIT not a blocker.

Reply:
We're already using same code for different errors. So I'd prefer another PR to focus on that (if necessary)